aidial_assistant/utils/log_config.py

```diff
@@ -5,7 +5,7 @@ def get_log_config(log_level: str) -> dict:
         "formatters": {
             "default": {
                 "()": "uvicorn.logging.DefaultFormatter",
-                "fmt": "%(levelprefix)s | %(asctime)s | %(name)s | %(process)d | %(message)s",
+                "fmt": "%(levelprefix)s | %(asctime)s | %(name)s | %(process)d | [trace_id=%(otelTraceID)s span_id=%(otelSpanID)s] - %(message)s",
```

Comment:
Make it dependent on logging flag from `TracingConfig`:

```
def get_log_config(log_level: str, otel_logging: bool) -> dict:
```

Reply:
Done.